platform/core/src/services/DisplaySetService/DisplaySetService.ts

```diff
@@ -204,7 +204,7 @@ export default class DisplaySetService extends PubSubService {
     }
 
     // If array of instances => One instance.
-    const displaySetsAdded = [];
+    const displaySetsAdded = [] as DisplaySet[];
```

Comment:
I generally prefer the following:
new Array<DisplaySet>()
to the casting.  Not sure which form we should use.